src/mmw/js/src/app.js

```diff
@@ -3,6 +3,8 @@
 var $ = require('jquery'),
     _ = require('lodash'),
     Marionette = require('../shim/backbone.marionette'),
+    settings = require('./core/settings').getSettings(),
+    itsi = require('./core/itsiEmbedInterface'),
```

Comment:
I'm open to suggestions for names better than `itsiEmbedInterface`. The PascalCasing looks weird and unwieldy but I couldn't come up with anything shorter that wasn't confusing.